sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -489,13 +489,16 @@ class CudfM2 extends CudfAggregate {
 class CudfMergeM2 extends CudfAggregate {
   override lazy val reductionAggregate: cudf.ColumnVector => cudf.Scalar =
     (col: cudf.ColumnVector) => {
-      withResource(col.copyToHost()) { hcv =>
+      val hcv = withResource(new NvtxRange("device-to-host", NvtxColor.ORANGE)) { _ =>
```

Comment:
I think the idea was to put the whole body of this reduction in a range, that way we can account for all of the time it takes (including the cpu loop)

Reply:
Whole body is already wrapped in another `NvtxRange`: https://github.com/NVIDIA/spark-rapids/blob/54f77b084a6b23b68af5e980d0161cf8f4ef1f1a/sql-plugin/src/main/scala/com/nvidia/spark/rapids/aggregate.scala#L320

Reply:
Anyway I'll put whole this body in `NvtxRange` too. That top level `NvtxRange` is for all reduction ops.